docs/SNMP_Alarms.md

```diff
@@ -7,20 +7,52 @@ monitoring service.
 
 Error indications come in two forms:
 
-*   For clearly-defined errors not based on thresholds, the Clearwater node sets an
-    ITU Alarm MIB from [RFC 3877](http://tools.ietf.org/html/3877) and sends an SNMP
-    notification to a configured external SNMP manager.
+*   For clearly-defined errors not based on thresholds, the Clearwater node sends an SNMP
+    notification to a configured external SNMP manager. (This notification can
+    be in one of two fomats, enterprise-specific or RFC 3877 - see below for
+    details.)
 
 *   For errors based on a threshold set on a statistic (such as latency targets or
     number of failed connections), the Clearwater node [exposes that statistic over
     SNMP](Clearwater_SNMP_Statistics.md). A downstream statistics aggregator from the Management and
     Orchestration (MANO) layer monitors these statistics, compares them to its
     configured thresholds, and raises alarms on that basis.
 
-## EMS
+This document focuses on the SNMP notifications we generate - threshold alarms are the responsibility of something external.
 
-To integrate with an EMS, the EMS must support the following capabilities of RFC 3877 to
-obtain the brief description, detailed description, and severity for the alarm:
+## Alarm Configuration
+
+There are two configuration options for alarms:
+
+- `snmp_ip` - this determines where alarms get sent, and is a comma-separated list of IP addresses (e.g. `snmp_ip=0.0.0.0,0.0.0.1`).
```

Comment:
I don't like this repeated section - we'll never remember to fix this up if it changes. I don't think you need alarm configuration in this doc